docs/docs/dev/api.md

```diff
@@ -394,38 +461,51 @@ Empty
       {
          "category_description": "",
          "category_name": "category1",
-         "category_id": "0"
+         "category_id": "0",
+         "color": "amber",
+         "deleted": true,
       },
       {
          "category_description": "",
          "category_name": "category2",
-         "category_id": "1"
+         "category_id": "1",
+         "color": "amber",
+         "deleted": false,
       }
    ]
 }
 ```
 
+:::
+::::
+
 ## Label Management
 
 Endpoints related to user-provided labels.
 
 ### Set element label
 
-Update the label of a selected element. This endpoint either sets or removes the label for a given element, depending on the `value` field. If `value == 'none'`, the element's label will be removed. Otherwise, the element will be assigned a bolean label depending on the string value ("true" -> True and "false" -> False. The `update_counter` determines whether the label changes will be reflected in the label change counters of the selected category (which may trigger the training of a new model). The parameter should be set to True, except for when labeling elements for precision evaluation.
+Update the label of a selected element. This endpoint either sets or removes the label for a given element.
 
----
+- In binary workspaces, the `binary_label` is set to the `category_id`. If `binary_label == 'none'`, the element's label will be removed. Otherwise, the element will be assigned a boolean label depending on the string value ("true" -> True and "false" -> False).
```

Comment:
```suggestion
- In binary workspaces, the value `binary_label` is set as the label of the element for the category with ID `category_id`. If `binary_label == 'none'`, the element's label will be removed. Otherwise, the element will be assigned a boolean label depending on the string value ("true" -> True and "false" -> False).
```